sdk/eventgrid/azure-eventgrid/azure/eventgrid/_helpers.py

```diff
@@ -110,3 +110,12 @@ def _eventgrid_data_typecheck(event):
     if isinstance(data, six.binary_type):
         raise TypeError("Data in EventGridEvent cannot be bytes. Please refer to"\
             "https://docs.microsoft.com/en-us/azure/event-grid/event-schema")
+
+def _decode(content):
+    try:
+        return base64.b64decode(content.encode('utf-8'))
```

Comment:
dumb question: what does the `content` look like, utf-8 base64 string?

Reply:
"data_base64":'Y2xvdWRldmVudA==',

Reply:
so we decided not to expose data_base64 to users at all 
see https://github.com/Azure/azure-sdk-for-python/pull/16661